### PR TITLE
Centralize GraphQL queries

### PR DESCRIPTION
**Pull request by rschlaefli**
GENERAL
- Move all queries from `TagList`, `QuestionList` and `SessionList` to a centralized file `queries.js`
- Prepare a file for mutations `mutations.js`
- Prepare queries (drafts) for several components that are not yet implemented

### CODE REVIEW

**Comment by rschlaefli** [comment]
[GraphQL Queries erarbeiten](https://trello.com/c/uOYYabMo/270-graphql-queries-erarbeiten)

**Comment by rschlaefli** [comment]
Will be merged after pending review.

File: src/queries/queries.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { gql } from 'react-apollo'
```
Comment (pascalzehnder) [inline src/queries/queries.js]: How shall we sort these queries? Did you sort them according to categories? As I see you didn't sort them in alphabetical order. Maybe we should name them as their name express the component in which it is used.

File: src/queries/queries.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        versions {
+          id
+          createdAt
+        }
+        createdAt
+        updatedAt
+      }
+  }
+`
+const QrGeneratorQuery = gql`
```
Comment (pascalzehnder) [inline src/queries/queries.js]: According to my first comment: To which component does this query belong?

**Review by pascalzehnder — APPROVED** [review]
I like having the GraphQL queries in a centralised file. As the amount of the queries is increasing, we need to pay attention having a predefined order of the queries.

**Pull request merged**